src/ripple/app/main/Application.h

```diff
@@ -137,6 +138,7 @@ class Application : public beast::PropertyStream::Source
     virtual TxQ&                        getTxQ() = 0;
     virtual ValidatorList&              validators () = 0;
     virtual ValidatorSite&              validatorSites () = 0;
+    virtual ValidatorKeys const&        validatorKeys () = 0;
```

Comment:
This seems unnecessary to me for this patch. See why further below.

Reply:
Done